app/services/data_sync/elasticsearch/user.rb

```diff
@@ -0,0 +1,51 @@
+module DataSync
+  module Elasticsearch
+    class User
+      RELATED_DOCS = %i[
+        articles
+        reactions
+        podcast_episodes
+        chat_channel_memberships
+        comments
+      ].freeze
+
+      SHARED_USER_FIELDS = %i[
+        username
+        name
+        pro
```

Comment:
`pro` is just an `attr_accessor` that I'm not sure it's even used:

```ruby
[6] pry(main)> $ u.pro

From: /devto/app/models/user.rb:26:
Owner: User
Visibility: public
Signature: pro()
Number of lines: 4

attr_accessor(
  :scholar_email, :new_note, :note_for_current_role, :user_status, :pro, :merge_user_id,
  :add_credits, :remove_credits, :add_org_credits, :remove_org_credits, :ghostify
)
```

`Pro` status is the result of the method `user.pro?` 

```ruby
[17] pry(main)> u.pro?
  ProMembership Load (9.8ms)  SELECT  "pro_memberships".* FROM "pro_memberships" WHERE "pro_memberships"."user_id" = $1 LIMIT $2  [["user_id", 11], ["LIMIT", 1]]
  Role Load (38.3ms)  SELECT "roles".* FROM "roles" INNER JOIN "users_roles" ON "roles"."id" = "users_roles"."role_id" WHERE "users_roles"."user_id" = $1 AND (((roles.name = 'pro') AND (roles.resource_type IS NULL) AND (roles.resource_id IS NULL)))  [["user_id", 11]]
=> false
```


and `.pro` won't appear in the `saved_changes`:

```ruby
[14] pry(main)> u.pro = "foobar"
=> "foobar"
[15] pry(main)> u.changes
=> {}
```